front-end/src/components/UI/Hamburger/Hamburger.jsx

```diff
@@ -1,12 +1,19 @@
-import React from 'react';
-import styles from "./Hamburger.module.css";
+import React, {useState} from 'react';
+import Lines from "./Lines/Lines";
+import NavBar from "./NavBar/NavBar";
+
+const Hamburger = () =>
+{
+    const [visibleNavBar, setVisibleNavBar] = useState(false)
+
+    if(visibleNavBar){
+
+    }
 
-const Hamburger = () => {
     return (
         <div>
-            <hr className={styles.hamburger} style={{marginTop: '1%'}}/>
-            <hr className={styles.hamburger} style={{marginTop: '2%'}}/>
-            <hr className={styles.hamburger}/>
+            <Lines visible={visibleNavBar} callback={setVisibleNavBar}/>
```

Comment:
Does make sense to remove `visible={visibleNavBar}` and use `callback={() => setVisibleNavBar(!visibleNavBar)}` ?